Assets/Tests/EditModeTests/CommunicationZoneVerticesTests.cs

```diff
@@ -239,11 +239,10 @@ public void TestCommunicationZoneVertices_WithWalls_BlockingCommunication()
 
             // Assert
             // Calculate intersection size
-            var intersection = new HashSet<int>(communicationZoneVertices.CommunicationZoneTiles[0]);
-            intersection.IntersectWith(communicationZoneVertices.CommunicationZoneTiles[1]);
+            var intersection = communicationZoneVertices.CommunicationZoneTiles[0].Intersect(communicationZoneVertices.CommunicationZoneTiles[1]);
 
             // Verify communication zones don't intersect because of the wall
-            Assert.AreEqual(0, intersection.Count);
+            Assert.AreEqual(0, intersection.Count());
```

Comment:
Any is better. Also Count() uses the IEnumerable implementation which is slow.

Reply:
Done